gpytorch/lazy/kronecker_product_added_diag_lazy_tensor.py

```diff
@@ -71,11 +65,11 @@ def _solve(self, rhs, preconditioner=None, num_tridiag=0):
     def _root_decomposition(self):
         evals, q_matrix = self.lazy_tensor.symeig(eigenvectors=True)
         updated_evals = DiagLazyTensor((evals + self.diag_tensor.diag()).pow(0.5))
```

Comment:
```suggestion
        updated_evals = DiagLazyTensor((evals + self.diag_tensor.diag()).clamp(min=0.).pow(0.5))
```

Reply:
I guess `diag_tensor` is assumed to be non-negative (should maybe check?) and `evals` should already be clamped in https://github.com/cornellius-gp/gpytorch/blob/master/gpytorch/lazy/lazy_tensor.py#L1881.